apps/condo/domains/common/components/icons/Icons.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Icon from '@ant-design/icons'
+
+
+const UserIconSVG: React.FC = () => {
```

Comment:
probably it's better to move it inside `user` domain

Reply:
And use UserIcon.jsx filename

Reply:
@AntonAL suggested to group all custom icons here, so idk...

Reply:
ok, could anyone add it as rule in our code style?

Reply:
> And use UserIcon.jsx filename

done 👍